src/python/visclaw/frametools.py

```diff
@@ -1144,17 +1242,6 @@ def printframes(plotdata=None, verbose=True):
     return plotdata
     # end of printframes
 
```

Comment:
I am fine with deleting this at it has been deprecated for awhile but we should check with everyone to make sure that this is really the case.